### PR TITLE
Budget-app 

### PR DESCRIPTION
**Pull request by mercymugambi**
## Below were the requirements for this milestone

Use ruby gems as software packages system.
Install Ruby on Rails framework.
Understand Rails RESTful design and router.
Use controllers to handle requests and render empty views.
Use params from browser request in a safe way.
Use preprocessed html file with embedded Ruby code.
Use layouts and templates for shared content.
Use database migration files to maintain database schema.
Use validations for models.
Secure app from n+1 problems.
Understand what ORM is.
Write SQL queries with ActiveRecord.
Set up associations between models.
Build a webapp that requires the user to log in.
Use devise gem for authentication.
Limit access to webapp resources based on authorization rules.
Analyze in writing why you have made a coding choice using one structure over another.

### CODE REVIEW

File: app/views/payments_entities/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  <div class="category-card">
+    <p><strong>Name:</strong> <%= pe.name %></p>
+    <p><strong>Amount:</strong>$<%= pe.amount %></p>
+  </div>
+<% end %>
+<% end %>
+
+<div class='bottom-button sign-links'>
+ <%= link_to 'Add New Transaction', new_group_payments_entity_path, class: 'new-cart a-tag' %>
+</div>
```
Comment (Munsa1) [inline app/views/payments_entities/index.html.erb]: - [x] Good job on creating the app, however kindly ensure your design follows the given design.

Your design
image

Given Design
image

- Kindly ensure that the items are aligned to the left and the price is aligned to the right for a much better design as provided in the given design.
- Also note that the price is bold text while the item name is lighter

File: app/assets/stylesheets/application.css [hunk 2]
```diff
@@ -13,3 +13,85 @@
  *= require_tree .
  *= require_self
  */
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@400;500;600;700&family=Roboto+Flex:wght@300;400;500;600;700&family=Roboto+Serif:wght@400;500;600;700&family=Roboto:wght@300;400;500;700&display=swap');
+@import 'font-awesome';
+
```
Comment (Munsa1) [inline app/assets/stylesheets/application.css]: - [ ] The provided font is `Proxima Nova` for this design. Kindly use it so as to match the design. If you're having trouble finding it online [here is a free link](https://freefontsfamily.com/proxima-nova-font-family-free-download/)

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @mercymugambi, 
required changes

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Suggested changes
_Check the comments under the review._

- [x] The video presentation is clear and shows the features of the project, kudos!

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by nahnahdev — APPROVED** [review]
## Project Status: APPROVED :green_circle: 
Hi @mercymugambi , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights :fire: 
- All requested changes implemented :heavy_check_mark: 
- Design modified :heavy_check_mark: 
- Recommended font used :heavy_check_mark: 
- All project requirements met :heavy_check_mark: 
- App works seamlessly as expected  :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**